moduleroot/.github/workflows/pr_test.yml.erb

```
@@ -134,7 +134,14 @@ jobs:
         echo
         echo ::endgroup::
         echo ::group::=== INVENTORY ===
-        sed -e 's/password: .*/password: "[redacted]"/' < inventory.yaml || true
+        if [ -f 'spec/fixtures/litmus_inventory.yaml' ];
+        then
+          FILE='spec/fixtures/litmus_inventory.yaml'
+         elif [ -f 'inventory.yaml' ];
```

Just minor alignment issue:
```suggestion
        elif [ -f 'inventory.yaml' ];
```